change/@ni-nimble-components-509eb87e-ffab-4277-bb4c-f9af5cd13a38.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Please update the Testing section of the PR description to include context about automated testing; there should always be justification if something is only being manually tested.

In this case it looks like you approved a Chromatic diff so you could mention that and note that this shows it was covered by existing auto tests.